tutorials/short-turnaround-time/content.adoc

```diff
@@ -0,0 +1,56 @@
+= How to develop with Flow, without Servlet-Container restarts
+
+:title: How to develop with Flow, without Servlet-Container restarts
+:type: text
+:author: Sven Ruppert
+:tags: HotSwapAgent, Hotswap, DCEVM, Flow, Java
+:description: How to develop with Flow, without Servlet-Container restarts
+:imagesdir: ./images
+
+== Why?
+Vaadin Flow is based on Java and a Servlet-Container.
+To see results, you need to deploy the Flow app, and the Servlet-Container must run.
+This startup will take a few seconds based on your machine and project.
+However, how could we get this turnaround time reduced?
+What should we do, to speed up the development?
+The answer is called DCEVM and Hotswap.
+
+== What is DCEVM?
+DCEVM is an extension to the JVM that will allow us to reload classes at runtime more easily.
+It is based on the OpenJDK version 8 and 11 so far. Maybe, more versions are available in the future.
+Check the DCEVM-website for this http://dcevm.github.io/[http://dcevm.github.io/]
+To use this, you need to download the version you need (version 8 or 11) and start the installer with the command.
+*java -jar installer.jar*
+Follow the instructions that the UI will give you. To make it easier to switch between
+a JDK with and without DCEVM you should use a dedicated installation of the JDK for this.
+
+== Hotswap
+After this is done, we need the second part called *HotSwapAgent*
+http://hotswapagent.org/[http://hotswapagent.org/].
+In contrast to standard Java, where the hotswap is limited to in-body code changes, the DCEVM + HotswapAgent allow following code changes:
+
+* Add/remove/modify class fields.
+* Add/remove/modify methods. Add/remove/modify method annotations
+* Add/remove/modify classes including anonymous classes. HotswapAgent handless correct anonymous class redefinitions.
+* Add/remove a static member of classes. HotswapAgent handles static member initialization.
+* Add/remove enum values
+* Refresh framework and application server settings
+The only unsupported operation is hierarchy change (change the superclass or remove an interface).
+DCEVM realizes hotswap on JVM level.
+HotwapAgent does the same on the level of Java frameworks and servlet containers.
+
+To activate HotSwapAgent, download the binary for your JDK (version 8 or 11 is supported right now) and
+enable this agent during the start of the JVM. *-javaagent:hotswap-agent.jar*
```

Comment:
How do I do this? Maybe you could give examples with some IDEs to make it easier for beginners